### PR TITLE
kev/74_fix_unused_code

### PR DESCRIPTION
**Pull request by junhaow1**
# Pull Request Details

## What issue does this PR address

- [A one line description of this PR]

- Link to associated issue: #

## Checklist

Complete the check-list below to ensure your branch is ready for PR.

Flutter Style Guide: https://survivor.togaware.com/gnulinux/flutter-style.html

- [x] Screenshots included in linked issue
- [x] Changes adhere to the team style and coding guideline
- [x] No confidential information
- [x] No duplicated content
- [ ] No lint check errors related to your changes (`make prep`)
- [ ] Pre-exisiting lint errors noted: 
- [x] Tested on at least one device
  - [ ] Android Phone
  - [ ] Android Emulator
  - [x] Chrome
  - [ ] iOS
  - [ ] Linux
  - [x] MacOS
  - [ ] Windows
- [ ] Added 2 reviewers

## Finalising

Once PR discussion is complete and 2 reviewers have approved:

- [ ] Merge dev into the branch
- [ ] Resolve any conflicts
- [ ] Bump appropriate version and build number in pubspec.yaml
- [ ] Add one line summary into CHANGELOG.md
- [ ] Push to git repository and review
- [ ] Merge PR into dev

### CODE REVIEW

File: lib/src/screens/initial_setup/gen_file_body.dart [hunk 1]
```diff
@@ -157,25 +156,28 @@ String genPubKeyFileBody(String resUrl, String pubKeyStr) {
   return keyFileBody;
 }
 
-/// Generates a profile file body in a predefined format.
-///
-/// This function takes two maps, `profData` and `authData`, as inputs. The `profData`
-/// map contains user profile information such as name and gender. The `authData` map
-/// includes authentication information, specifically an access token.
+// comment out the following function as it is not used in the current version
+// of the app, anushka might need to use to in the future so keeping it here.
```
Comment (junhaow1) [inline lib/src/screens/initial_setup/gen_file_body.dart:160]: Hi @anushkavidanage , could you please have a look at the following part, if it is not needed, I can delete them. If we need to use them in the future, we can comment out just like what I did. :)

File: lib/src/screens/initial_setup/widgets/initial_setup_welcome.dart [hunk 2]
```diff
@@ -94,80 +84,86 @@ SizedBox initialSetupWelcome(BuildContext context) {
   );
 }
 
-/// Creates a row widget displaying a piece of profile information.
-///
-/// This function constructs a `Row` widget designed to display a single piece
-/// of information in a profile UI. It is primarily used for laying out text-based
-/// information such as names, titles, or other key details in the profile section.
+// comment out the following function as it is not used in the current version
+// of the app, anushka might need to use to in the future so keeping it here.
```
Comment (junhaow1) [inline lib/src/screens/initial_setup/widgets/initial_setup_welcome.dart:88]: Hi @anushkavidanage , could you please have a look at the following part, if it is not needed, I can delete them. If we need to use them in the future, we can comment out just like what I did. :)

File: lib/src/screens/initial_setup/widgets/initial_setup_welcome.dart [hunk 3]
```diff
@@ -94,80 +84,86 @@ SizedBox initialSetupWelcome(BuildContext context) {
...
-                ),
-        ],
-      ),
-      SizedBox(
-        height: screenHeight(context) * 0.005,
-      )
-    ],
-  );
-}
+// comment out the following function as it is not used in the current version
```
Comment (junhaow1) [inline lib/src/screens/initial_setup/widgets/initial_setup_welcome.dart:114]: Hi @anushkavidanage , could you please have a look at the following buildLabelRow widget , if it is not needed, I can delete them. If we need to use them in the future, we can comment out just like what I did. :)

File: lib/src/solid/api/rest_api.dart [hunk 4]
```diff
@@ -355,202 +356,211 @@ Map<dynamic, dynamic> generateDefaultFiles(String appName) {
...
+//   }
+
+//   if (createUpdateRes == 'ok') {
+//     return createUpdateRes;
+//   } else {
+//     throw Exception('Failed to create/update the shared file.');
+//   }
+// }
+
+// comment out the following function as it is not used in the current version
```
Comment (junhaow1) [inline lib/src/solid/api/rest_api.dart:518]: Hi @anushkavidanage , could you please have a look at the following initialProfileUpdate function , if it is not needed, I can delete them. If we need to use them in the future, we can comment out just like what I did. :)

**Comment by anushkavidanage** [comment]
@junhaow1 There are some conflicts after merging my branch. Could you please have a look and resolve those? I can then have a look at the remaining parts

**Comment by junhaow1** [comment]
> @junhaow1 There are some conflicts after merging my branch. Could you please have a look and resolve those? I can then have a look at the remaining parts

on it

**Comment by junhaow1** [comment]
Hi @anushkavidanage there is no conflict anymore :)

**Pull request closed without merging**